tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,46 @@
+*** Settings ***
+Resource    ../../../Resources/ODS.robot
+Resource    ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Resource    ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+
+*** Test Cases ***
+Any OpenShift User Should Also Be A RHODS User
+    [Documentation]    Any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       ODS-1164
+    Set Library Search Order    SeleniumLibrary   InstalledOperators
+    Open Browser    ${OCP_CONSOLE_URL}    ${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+    Navigate To Page  Workloads  ConfigMaps
+    Select Project By Name    redhat-ods-applications
+    Select Rhods-Groups-Config
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Spawn Notebook
+
+
+*** Keywords ***
+Select Rhods-Groups-Config
+    [Documentation]    Select rhods-groups-config and click
+    Wait Until Page Contains Element    xpath=//input[@class='pf-c-form-control co-text-filter']
+    Input Text    xpath=//input[@class='pf-c-form-control co-text-filter']    rhods-groups-config
```

Comment:
sorry I didn't remember earlier, but there is a keyword to search inside openshift table which would replace line 25 and 26. Could you try using it? `Search Last Item Instance By Title in OpenShift Table` located in `Resources > Page > OCPDashboard`

Reply:
''Search Last Item Instance By Title in OpenShift Table'' is checking if the page contains the search item before entering the text in the search box, But 'rhods-group-config' cannot be found unless scrolled down or searched. So it is failing there.

Reply:
hm yeah I think it's a bug of the keyword. I think it can be solved by moving the check about "page contains the search item" after the input text in the search box.

Reply:
Changed it.